src/geo_interface.jl

```diff
@@ -39,21 +43,47 @@ function GeoInterface.getgeom(
     getGeometry(geom, i)
 end
 GeoInterface.getgeom(::MultiPointTrait, geom::MultiPoint, i) = getGeometry(geom, i)::Point
+GeoInterface.getgeom(::MultiPointTrait, geom::MultiPoint) = getGeometry(geom, i)::Point
 GeoInterface.getgeom(::MultiLineStringTrait, geom::MultiLineString, i) =
     getGeometry(geom, i)::LineString
 GeoInterface.getgeom(::MultiPolygonTrait, geom::MultiPolygon, i) =
     getGeometry(geom, i)::Polygon
-GeoInterface.getgeom(
-    ::Union{LineStringTrait,LinearRingTrait},
-    geom::Union{LineString,LinearRing},
-    i,
-) = getPoint(geom, i)
+function GeoInterface.getgeom(::AbstractGeometryTrait, geom::Union{LineString,LinearRing}, i)
+    refs = Ref{Float64}(), Ref{Float64}(), Ref{Float64}() # 3 Refs is faster than a Vector
```

Comment:
Cool, might be good to update/merge with the getCoord method used elsewhere as well.

Reply:
Have to leave that for another time